modules/videoio/src/cap_mjpeg_encoder.cpp

```diff
@@ -268,7 +268,7 @@ class mjpeg_buffer_keeper
             m_buffer_list[0].finish();
 
             m_data_len = m_buffer_list[0].get_len();
-            m_last_bit_len = m_buffer_list[0].get_bits_free() ? 32 - m_buffer_list[0].get_bits_free() : 0;
+            m_last_bit_len = 32 - m_buffer_list[0].get_bits_free();
```

Comment:
see https://github.com/opencv/opencv/blob/b3c896370cb6b61f9a45a1fa4dc2483a50d3097d/modules/videoio/src/cap_mjpeg_encoder.cpp#L192

Reply:
if you call mjpeg_buffer::finish() than mjpeg_buffer::get_len() returns the number of occupied integers and mjpeg_buffer::get_bits_free() returns the number of free bits in the last integer (from 0 to 31)